# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../inkscape/inkscape-0.46-gtkopen.patch
# Copyright (C) 2008 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

------------------------------------------------------------------------
r18721 | buliabyak | 2008-05-24 21:32:40 +0200 (Sat, 24 May 2008) | 1 line

this hack does not seem to be needed with modern gtk, and it prevents remembering of last-browsed directory from working

Index: src/ui/dialog/filedialogimpl-gtkmm.cpp
===================================================================
--- ./src/ui/dialog/filedialogimpl-gtkmm.cpp	(revision 18720)
+++ ./src/ui/dialog/filedialogimpl-gtkmm.cpp	(revision 18721)
@@ -761,10 +761,6 @@
 bool
 FileOpenDialogImplGtk::show()
 {
-    Glib::ustring s = Glib::filename_to_utf8 (get_current_folder());
-    if (s.length() == 0)
-        s = getcwd (NULL, 0);
-    set_current_folder(Glib::filename_from_utf8(s)); //hack to force initial dir listing
     set_modal (TRUE);                      //Window
     sp_transientize((GtkWidget *)gobj());  //Make transient
     gint b = run();                        //Dialog
